google-cloud-storage/src/main/java/com/google/cloud/storage/GrpcConversions.java

```diff
@@ -63,6 +66,19 @@ final class GrpcConversions {
   private final Codec<Integer, String> crc32cCodec =
       Codec.of(this::crc32cEncode, this::crc32cDecode);
 
+  @VisibleForTesting
+  final Codec<OffsetDateTime, Timestamp> timestampCodec =
```

Comment:
Why isn't this in our codecs source instead?

Reply:
Following the pattern from ApiaryConversions for that conversion to DataTime[1].

Give this conversion is specific to the use of protos it probably makes sense to keep it near the proto conversions. Or, is there another location you'd like to see this moved to?

[1] https://github.com/googleapis/java-storage/blob/359504b1b8ca4f15b13106cfbd0440cc22598511/google-cloud-storage/src/main/java/com/google/cloud/storage/ApiaryConversions.java#L131-L152